share/spack/gitlab/cloud_pipelines/configs/cray-rhel/x86_64_v3/packages.yaml

```diff
@@ -0,0 +1,5 @@
+packages:
+  all:
+    target: ["x86_64_v3"]
+    require:
+    - "target=x86_64_v3"
```

Comment:
do we need both the target and require lines here? seems like just require would be fine.